### PR TITLE
[FIX] project: fix various bugs in v16.0

### PR DESCRIPTION
**Pull request by kcv-odoo**
This PR fixed various bugs in v16.0 .

task-2999323

### CODE REVIEW

File: addons/project/models/project.py [hunk 1]
```diff
@@ -388,10 +388,7 @@ def _read_group_stage_ids(self, stages, domain, order):
 
     # Not `required` since this is an option to enable in project settings.
     stage_id = fields.Many2one('project.project.stage', string='Stage', ondelete='restrict', groups="project.group_project_stages",
-        tracking=True, index=True, copy=False, default=_default_stage_id, group_expand='_read_group_stage_ids',
-        help="The step, or stage, that your project is in. You can add, remove, and move stages around to adapt the pipeline to your own needs.\n"
-            "Via the stage configuration you can set up email templates and automate the sending of emails when a project moves into the stage.\n"
-            "Projects in a folded stage are considered as closed.")
+        tracking=True, index=True, copy=False, default=_default_stage_id, group_expand='_read_group_stage_ids')
```
Comment (xavierbol) [inline addons/project/models/project.py:391]: Don't we have to remove the text in the `.pot` and `.po` file? 🤔

File: addons/project/views/project_views.xml [hunk 2]
```diff
@@ -681,6 +681,7 @@
             <field name="model">project.project</field>
             <field name="arch" type="xml">
                 <tree decoration-muted="active == False" string="Projects" multi_edit="1" sample="1">
+                    <field name="company_id" invisible="1"/>
```
Comment (xavierbol) [inline addons/project/views/project_views.xml:684]: Can you add the company_id field below the other `company_id` one? 
It is to keep selecting the right company_id with an eventual xpath. 🙂 

Thanks in advance! 🙂

File: addons/project/static/src/js/project_form.js [hunk 3]
```diff
@@ -172,3 +172,6 @@ export const ProjectFormView = FormDescriptionExpanderView.extend({
 viewRegistry.add('project_task_form', ProjectFormView);
 
 viewRegistry.add('form_description_expander', FormDescriptionExpanderView)
+
+// TODO: remove me in a master and create a new project view without js_class
+viewRegistry.add('project_form', FormDescriptionExpanderView);
```
Comment (xavierbol) [inline addons/project/static/src/js/project_form.js:177]: Is it not fixed with this PR: https://github.com/odoo/enterprise/pull/32188 ? 🤔

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I have made some comments. 🙂

File: addons/project/static/src/js/project_form.js (hunk 3 above)
Comment (kcv-odoo) [reply addons/project/static/src/js/project_form.js]: Yes, It is fixed by that PR :star_struck: and Done the other changes. Thanks :slightly_smiling_face:

**Review by xavierbol — APPROVED** [review]
LGTM, thanks again for your work! 🙂

**Review by IT-Ideas — APPROVED** [review]
Thanks for your work 🙂

**Comment by IT-Ideas** [comment]
@robodoo r+

**Comment by IT-Ideas** [comment]
@robodoo rebase-ff

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**